DominicGBauer/information-access
# fix: page updates bug

# Pull Request

## Scope
Partial routing fixes.

## Work Done
* Page resets to 1 after year selection
* preview pages can be directly accessed via URL

## Steps to Test
```npm run start```
* Search and move to different page and then select a year and test for custom year as well. URL should reflect page 1. 
* For preview, navigate to a preview page, copy the URL and paste in a new page and run; should return same results.

## Screenshot
![image](https://user-images.githubusercontent.com/80747408/138067980-3d5f0bd1-6be6-4493-a069-4c57b3745d51.png)
![image](https://user-images.githubusercontent.com/80747408/138068264-0601a8fd-806f-497e-ad96-bd1ff6ca7aff.png)
